api/types.go

```diff
@@ -0,0 +1,11 @@
+package api
```

Comment:
I think our conclusion was to add the types into `skipper` and `routesrv` packages, because we want to be client/server independent of each other.